src/components/Button.js

```diff
@@ -2,14 +2,30 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = props => {
-  const { name } = props;
+  const { name, wide, color } = props;
+  const styles = {
```

Comment:
- [ ] Buttons should take with of 25% except the "0" button. Add a styling for that.